### ISSUE TITLE
HLS Loader Extension Not Working

### ISSUE THREAD

**Issue by simonferndriger** [brave/brave-browser#19755]
## Description 
The [HLS Loader](https://www.hlsloader.com/) extension does not work with Brave. The extension can be installed, loaded and activated, but the streams cannot be recorded. The extension works flawlessly with Google Chrome Canary.

## Steps to Reproduce
   1. Install HLS Loader extension
   2. Visit https://streamdiver.com/demo/
   3. Click the extension to start recording

## Actual result:
No recording is started (normal mode is not working:
![image](https://user-images.githubusercontent.com/1435963/143568453-f65e79a0-a1b2-4498-91de-a65dce0f33ec.png)

## Expected result:
However, the normal stream recording should work, since it does on the mentioned page with Chrome Canary.

## Reproduces how often: 
Reproduction always works.

## Brave version (info found on brave://version)
1.32.113 Chromium: 96.0.4664.45 (Official Build) (64-bit)

**Comment by rebron** [comment]
@simonferndriger Did you contact the developers for HLS Loader extension as well? We're likely going to need the developers there to address this issue and support us.

**Comment by simonferndriger** [comment]
No, I didn't. But you can reach him at loadmonkey.inquiry@gmail.com
